Report creation from cmd line can fail sometimes

Reported by Carlos Rodrigo Souza Santos. Checking the log I found:
```
Caused by: java.lang.IllegalArgumentException: Inconsistency of field data structures across documents for field [office:extended-properties:TotalTime:string] of doc [11]. doc values type: expected 'SORTED_NUMERIC', but it has 'SORTED_SET'.
                at org.apache.lucene.index.IndexingChain$FieldSchema.raiseNotSame(IndexingChain.java:1357)
                at org.apache.lucene.index.IndexingChain$FieldSchema.assertSame(IndexingChain.java:1352)
                at org.apache.lucene.index.IndexingChain$FieldSchema.setDocValues(IndexingChain.java:1393)
                at org.apache.lucene.index.IndexingChain.updateDocFieldSchema(IndexingChain.java:819)
                at org.apache.lucene.index.IndexingChain.processDocument(IndexingChain.java:601)
                at org.apache.lucene.index.DocumentsWriterPerThread.updateDocuments(DocumentsWriterPerThread.java:241)
                at org.apache.lucene.index.DocumentsWriter.updateDocuments(DocumentsWriter.java:432)
                at org.apache.lucene.index.IndexWriter.updateDocuments(IndexWriter.java:1532)
                at org.apache.lucene.index.IndexWriter.addDocuments(IndexWriter.java:1503)
                at iped.engine.task.index.IndexTask.process(IndexTask.java:148)
```
Although this is a regression, I think it is not critical, since report creation from UI works.

--- a/iped-engine/src/main/java/iped/engine/task/index/IndexTask.java
+++ b/iped-engine/src/main/java/iped/engine/task/index/IndexTask.java
@@ -254,7 +254,11 @@ public class IndexTask extends AbstractTask {
             }
         }
 
-        IndexItem.loadMetadataTypes(new File(output, "conf")); //$NON-NLS-1$
+        // Don't load default types if generating report, they will be loaded later.
+        // See https://github.com/sepinf-inc/IPED/issues/1258
+        if (!caseData.isIpedReport()) {
+            IndexItem.loadMetadataTypes(new File(output, "conf")); //$NON-NLS-1$
+        }
         loadExtraAttributes();
 
         this.autoParser = new StandardParser();

